src/dotnet-install.ps1

```diff
@@ -33,7 +33,10 @@
     - 3-part version in a format A.B.C - represents specific version of build
           examples: 2.0.0-preview2-006120, 1.1.0
 .PARAMETER Internal
-    Set this switch to download internal builds. It is necessary to use FeedCredential option with this.
+    Download internal builds. Requires providing credentials via -FeedCredential parameter.
+.PARAMETER FeedCredential
+    Token to access Azure feed. Used as a query string to append to the Azure feed.
+    It allows changing the URL to use non-public blob storage accounts. This parameter typically is not specified.
```

Comment:
This one is a reference to `AzureFeed` option. You change your feed to something else using `AzureFeed`, then you set `FeedCredential` to get access to that feed.

Though I agree that it can use a bit of clarification. Maybe something like:
```
It allows access to the private feed specified by "-AzureFeed" option in the case that the default feed is not used.
```

Reply:
I am not sure I would like to do more rearrangements in the parameters order, there is also `UncachedFeed` that we would like to have near `AzureFeed`. Maybe this might lead to even more problems with order.